src/utils/raw.rs

```diff
@@ -352,7 +352,7 @@ pub mod windows {
     pub fn to_u16s<S: AsRef<OsStr>>(s: S) -> io::Result<Vec<u16>> {
         fn inner(s: &OsStr) -> io::Result<Vec<u16>> {
             let mut maybe_result: Vec<u16> = s.encode_wide().collect();
-            if maybe_result.iter().any(|&u| u == 0) {
+            if memchr_u16(0, &maybe_result).is_some() {
```

Comment:
I don't think we need this bit - adding in a FIXME  (in the other hunk) in our code base for an unmeasured performance change in a non-inner-loop(*) part of the code  doesn't make sense to me. The rest of the patch looks great.

*: this is only called from symlink_dir, which I don't have exact figures on, but I am reasonably sure its at most O(dists). so maybe 10 times a process run? That 11% improvement is going to be maybe 0.0001 of runtime. If that.

Reply:
That's fair. I removed that commit.